ReviewerUI/src/js/components/Layout.js

```diff
@@ -0,0 +1,29 @@
+import React from "react/lib/React";
+
+import Header from "./Header";
+import SearchCriteriaForm from "./SearchCriteriaForm";
+import SearchResultsForm from "./SearchResultsForm";
+
+export default class Layout extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      client: "table-next",
+      product: "metal-table"
+    };
+  }
+
+  handleResult(client, product) {
+    this.setState({client, product});
+  }
+
+  render() {
+    return (
+      <div class="container">
+        <Header />
+        <SearchCriteriaForm onUpdate={this.handleResult.bind(this)} />
```

Comment:
Lets have components listen to changes and leave this object just as a container